e2e/app-with-domain/app_with_domain_test.go

```diff
@@ -70,47 +70,64 @@ var _ = Describe("App With Domain", func() {
 		})
 	})
 
-	Context("when initializing a Load Balanced Web Service", func() {
+	Context("when initializing Load Balanced Web Services", func() {
 		var svcInitErr error
 
-		BeforeAll(func() {
+		It("svc init should succeed for creating the hello service", func() {
 			_, svcInitErr = cli.SvcInit(&client.SvcInitRequest{
-				Name:       svcName,
+				Name:       "hello",
 				SvcType:    "Load Balanced Web Service",
-				Dockerfile: "./hello/Dockerfile",
+				Dockerfile: "./src/Dockerfile",
 				SvcPort:    "80",
 			})
+			Expect(svcInitErr).NotTo(HaveOccurred())
 		})
 
-		It("svc init should succeed", func() {
+		It("svc init should succeed for creating the frontend service", func() {
+			_, svcInitErr = cli.SvcInit(&client.SvcInitRequest{
+				Name:       "frontend",
+				SvcType:    "Load Balanced Web Service",
+				Dockerfile: "./src/Dockerfile",
+				SvcPort:    "80",
+			})
 			Expect(svcInitErr).NotTo(HaveOccurred())
 		})
 	})
 
 	Context("when deploying a Load Balanced Web Service", func() {
 		var deployErr error
 
-		It("svc deploy to test should succeed", func() {
+		It("deploy hello to test should succeed", func() {
 			_, deployErr = cli.SvcDeploy(&client.SvcDeployInput{
-				Name:     svcName,
+				Name:     "hello",
 				EnvName:  "test",
 				ImageTag: "hello",
 			})
 			Expect(deployErr).NotTo(HaveOccurred())
 		})
 
-		It("svc deploy to prod should succeed", func() {
+		It("deploy frontend to test should succeed", func() {
+			_, deployErr = cli.SvcDeploy(&client.SvcDeployInput{
```

Comment:
oh boy this test is gonna take forever to run :D

Reply:
about 1h in total